adapters/mqtt-vertx-base/src/test/resources/logback-test.xml

```diff
@@ -27,7 +27,7 @@
     <appender-ref ref="STDOUT" />
   </root>
 
-  <logger name="org.eclipse.hono.adapter" level="INFO"/>
+  <logger name="org.eclipse.hono.adapter" level="DEBUG"/>
```

Comment:
Do we want to leave this to DEBUG or is it just overlooked?

Reply:
No, this should be INFO ... will fix. Thanks for spotting :+1: